src/style.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir.ttc) format("truetype"),
+    url(/fonts/Avenir-Book.ttf) format("truetype"),
+    url(/fonts/Avenir-Heavy.ttf) format("truetype");
+}
+/* #region body */
+body {
+  font-family: "Avenir", sans-serif;
+  margin: 0;
+  padding: 0;
+}
+/* #endregion */
+
+/* #region general search style */
+.search {
+  width: auto;
+  margin: 20px 8px;
+  border: solid 1px #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+
+  display: flex;
+  align-items: center;
+  padding-left: 25px;
+}
+
+.icon__search {
+  height: 20px;
+  width: 20px;
+  margin-right: 17px;
+  background-image: url(/images/Search.svg);
+  background-size: contain;
+}
+
+.input__search {
+  border: 0 solid;
+  font-size: 16px;
+  font-weight: 800;
+  color: #3d4e61;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)